ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts

```diff
@@ -109,9 +113,26 @@ export class FeedconfigurationComponent implements OnInit {
         }
     }
 
+    private selectAllProcesses(idProcessGroup: string) {
+        for (let processId of this.processIdsByProcessGroup.get(idProcessGroup))
+            this.selectAllStates(processId);
+        this.isAllProcessesSelectedPerProcessGroup.set(idProcessGroup, true);
+    }
+
+    private unselectAllProcesses(idProcessGroup: string) {
+        for (let processId of this.processIdsByProcessGroup.get(idProcessGroup))
+            this.unselectAllStates(processId);
+        this.isAllProcessesSelectedPerProcessGroup.set(idProcessGroup, false);
+    }
+
```

Comment:
As per previous comment these 2 methods could be replaced by a single  method toggleSelectAllProcesses(idProcessGroup: string)

Reply:
Done